conf/modules/heli_throttle_curve.xml

```diff
@@ -4,6 +4,15 @@
   <doc>
     <description>Throttle Curve Mixers</description>
   </doc>
+  <settings>
+    <dl_settings>
+      <dl_settings NAME="ThrottleCurve">
+         <dl_setting var="throttle_curve.rpm_fb_p" min="0.00" step="0.005" max="100" shortname="rpm_fb_p" param="THROTTLE_CURVE_RPM_FB_P" />
+         <dl_setting var="throttle_curve.rpm_fb_i" min="0.00" step="0.005" max="100" shortname="rpm_fb_i" param="THROTTLE_CURVE_RPM_FB_I" />
+         <dl_setting var="throttle_curve.throttle_trim" min="-9600" step="0" max="9600" shortname="throttle_trim" />
```

Comment:
This trim was for our forward flight. (I removed throttle with this in a specific curve). See the other comment: "see here"